business_requirement_deliverable_cost/models/business.py

```diff
@@ -185,6 +185,29 @@ class BusinessRequirement(models.Model):
         'RL Total Cost',
         compute='_compute_rl_total_cost'
     )
+    total_revenue_ci = fields.Float(
+        compute='_compute_total_revenue_in_ci',
+        string='Total Revenue in CI',
+    )
+    ci_currency_id = fields.Many2one(
```

Comment:
`ci_` prefix would be means nothing as name for a field, better add what functional means on the `help` parameter

- [ ] Add `help` param with a short meaning
- [ ] Then name then should be `currency_id`
- [ ] Remove the readonly
- [ ] Should be a `default` instead of `compute` from the br.self.partner_id.property_product_estimation_pricelist.currency_id
but later the user what change then could.